csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -577,7 +577,7 @@ private void TestOverridableInitializerMetadata()
             }
         }
 
-        [Fact]
+        [SkipNonPackageTests]
```

Comment:
Custom op dll tests need not run during nuget package testing as the package doesn't contain the dll. The test will run during regular C# building and testing in the other job legs of the pipeline.